tutorials/intro/plot_introduction.py

```diff
@@ -1,404 +1,293 @@
+# :term:`SSP projectors <projector>` for environmental noise, and are discussed
+# in a later tutorial. In addition to the information displayed during loading,
+# you can get a glimpse of the basic details of a :class:`~mne.io.Raw` object
+# by printing it:
 #
-# Read data from file:
+# .. TODO edit prev. paragraph when projectors tutorial is added: ...those are
+#     discussed in the tutorial :ref:`projectors-tutorial`. (or whatever link)
 
-raw = mne.io.read_raw_fif(raw_fname)
 print(raw)
-print(raw.info)
 
-##############################################################################
-# Look at the channels in raw:
+###############################################################################
+# :class:`~mne.io.Raw` objects also have several built-in plotting methods; in
+# interactive Python sessions the basic :meth:`~mne.io.Raw.plot` method is
+# interactive and allows scrolling, scaling, bad channel marking, annotation,
+# projector toggling, etc.
 
-print(raw.ch_names)
+raw.plot(duration=5, n_channels=30)
 
-##############################################################################
-# Read and plot a segment of raw data
-
-start, stop = raw.time_as_index([100, 115])  # 100 s to 115 s data segment
-data, times = raw[:, start:stop]
-print(data.shape)
-print(times.shape)
-data, times = raw[2:20:3, start:stop]  # access underlying data
-raw.plot()
-
-##############################################################################
-# Save a segment of 150s of raw data (MEG only):
-
-raw.save('sample_audvis_meg_raw.fif', tmin=0, tmax=150, picks='meg',
-         overwrite=True)
-
-##############################################################################
-# Define and read epochs
-# ^^^^^^^^^^^^^^^^^^^^^^
+###############################################################################
+# Preprocessing
+# ^^^^^^^^^^^^^
 #
-# First extract events:
-
-events = mne.find_events(raw, stim_channel='STI 014')
-print(events[:5])
-
-##############################################################################
-# Note that, by default, we use stim_channel='STI 014'. If you have a different
-# system (e.g., a newer system that uses channel 'STI101' by default), you can
-# use the following to set the default stim channel to use for finding events:
-
-mne.set_config('MNE_STIM_CHANNEL', 'STI101', set_env=True)
-
-##############################################################################
-# Events are stored as a 2D numpy array where the first column is the time
-# instant and the last one is the event number. It is therefore easy to
-# manipulate.
+# MNE-Python supports a variety of preprocessing approaches and techniques
+# (maxwell filtering, signal-space projection, independent components analysis,
+# filtering, downsampling, etc), see the full list of functions in the
+# :mod:`mne.preprocessing` and :mod:`mne.filter` submodules. Here we'll quickly
+# clean up our data by adding SSP projectors for eye movements and heartbeats;
```

Comment:
...by creating spatial filters (SSP) for eye movements and heartbeats, and projecting these out of the raw data. We'll also plot the topographies associated with the projectors.